### PR TITLE
feat: deposit matched loans test removed from bp and checkout pages

### PR DESCRIPTION
**Pull request by roger-in-kiva**
Deposit matched loans test removed from borrower profile and checkout pages

### CODE REVIEW

File: src/pages/Checkout/CheckoutPage.vue [hunk 1]
```diff
@@ -73,7 +72,6 @@
 							:promo-fund="derivedPromoFund"
 							@refreshtotals="refreshTotals"
 							@updating-totals="setUpdatingTotals"
-							:show-matched-loan-kiva-credit="showMatchedLoanKivaCredit && requireDepositsMatchedLoans"
```
Comment (dyersituations) [inline src/pages/Checkout/CheckoutPage.vue:76]: Looks like `showMatchedLoanKivaCredit` computed can be removed

File: src/components/Checkout/OrderTotals.vue [hunk 2]
```diff
@@ -189,7 +185,6 @@ import { removeCredit } from '@/util/checkoutUtils';
 import showVerificationLightbox from '@/graphql/mutation/checkout/showVerificationLightbox.graphql';
 import KvTooltip from '@/components/Kv/KvTooltip';
 import VerifyRemovePromoCredit from '@/components/Checkout/VerifyRemovePromoCredit';
-import MatchedLoanKivaCredit from '@/components/Checkout/MatchedLoanKivaCredit';
```
Comment (dyersituations) [inline src/components/Checkout/OrderTotals.vue:192]: Looks like this component file can be deleted.

File: src/components/Checkout/LoanMatcher.vue [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <template functional>
 	<p class="tw-text-small tw-text-secondary matching-text">
 		<!-- eslint-disable-next-line max-len -->
-		<span :class="props.disableMatching && 'tw-line-through'">Matched by {{ props.matchingText }}</span>{{ props.disableMatching ? '. Loan matching only applies when you pay in full with new funds using Card, Paypal, Google Pay, or Apple Pay.' : '' }}
+		<span class="tw-line-through">Matched by {{ props.matchingText }}</span>
```
Comment (dyersituations) [inline src/components/Checkout/LoanMatcher.vue:4]: Do we want this `tw-line-through` class here? It looks like it was only added before when `props.disableMatching` was `true`.

File: src/pages/Checkout/CheckoutPage.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/pages/Checkout/CheckoutPage.vue:76]: Done

File: src/components/Checkout/OrderTotals.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/components/Checkout/OrderTotals.vue:192]: Done

File: src/components/Checkout/LoanMatcher.vue (hunk 3 above)
Comment (roger-in-kiva) [reply src/components/Checkout/LoanMatcher.vue]: even the span, thanks for your very sharp-eyed way!

**Review by dyersituations — APPROVED** [review]

**Pull request merged**